supporting_content/dimensions/listunitprice.md

```diff
@@ -0,0 +1,83 @@
+# Column: ListUnitPrice
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                     | Column                   |
+|:----------|:-----------------------------|:-------------------------|
+| AWS       | CUR;<br>AWS Price List API   | pricing/publicOnDemandCost: The total cost for the line item based on public On-Demand Instance rates. If you have SKUs with multiple On-Demand public costs, the equivalent cost for the highest tier is displayed. For example, services offering free-tiers or tiered pricing.<br>Since the pricing/publicOnDemandCost currently doesn't provide the correct price in all tier-based pricing use cases, we need to rely on the AWS Price List API. |
+| GCP       | BigQuery Pricing Data Export | Not available in BigQuery Billing Export, but can be resolved from the list_price Struct in the BigQuery Pricing Data Export |
+| Microsoft | Cost details;<br>Azure Retail Prices REST API | pay-as-you-goPrice/payGPrice: Retail price for the resource<br>Note: Identified some discrepancy between the value in Cost details and the Retail Price obtained from the Azure Retail Prices REST API, particularly in the context of CSPs. Might be related to Partner Billing UC? |
```

Comment:
The difference between the pricesheet and cost details is intentional and should be the difference between the pricing unit and usage unit. I don't think that's worth calling out in this table. It's fine to call it out in supporting content, but we should add context about why it's like that.

Reply:
**ASSUMPTIONS:**

I assumed that both Retail Prices API and Cost details provide PricingUnit and ListUnitPrice (per PricingUnit, not UsageUnit)

- Retail Prices API via retailPrice, unitOfMeasure
- Cost details via PayGPrice, unitOfMeasure

I anticipated alignment between the values but observed certain discrepancies and wasn't sure about the cause.


**SAMPLE DATA:**

- Retail Prices API response:

```
{
	"currencyCode": "USD",
	...
	"retailPrice": 0.00036,
	...
	"meterId": "b9e5e77c-a0b3-4a2c-9b8b-57fa54f31c52",
	...
	"unitOfMeasure": "10K",
	...
}
```

- Sample PAYG Cost details record
  - meterId: b9e5e77c-a0b3-4a2c-9b8b-57fa54f31c52
  - unitOfMeasure: 10K
  - PayGPrice: 0.00036

- Sample CSP Cost details record
  - meterId: b9e5e77c-a0b3-4a2c-9b8b-57fa54f31c5
  - unitOfMeasure: 10K
  - PayGPrice: 0.0003707


**QUESTION:** 

If the assumption regarding the Azure Cost Details PayGPrice is incorrect, **would it be acceptable to exclude the reference to PayGPrice and rephrase it as: _'...ListUnitPrice is not available in Cost Details, but can be obtained through the Retail Prices API...'_?**

Reply:
Discussed and decided to keep this note about discrepancy with PayGPrice in the supporting content.